### ISSUE TITLE
Sync already read articles also

### ISSUE THREAD

**Issue by xoxys** [readrops/Readrops#60]
I've connected my FreshRSS server to the app and had no unread articles at this time. I was a bit confused because Readrops has synced all my feeds but does not show any article. Even if I select "Show read articles" nothing happens. 

Is it intended? Are only new unread articles synced?

**Comment by Shinokuni** [comment]
Yes, it is intended behavior. When syncing for the first time, Readrops fetches only unread items.

**Comment by xoxys** [comment]
Thanks for your reply. Is this behavior discussable? :) IMO it's a bit uncommon in comparison to other readers and personally I would prefer syncing read and unread items until the configured "Maximum number of items per feed" is reached.

**Comment by deadwax** [comment]
> I would prefer syncing read and unread items until the configured "Maximum number of items per feed" is reached.

I second this, why is the first use treated differently to subsequent uses? It's the same user, presumably wanting the same behaviour.

**Comment by Shinokuni** [comment]
Here are my thoughts about that.

I used OCReader before developing Readrops which had the same behavior (actually I reused it).
> it's a bit uncommon in comparison to other readers

That is why I find this a little bit weird. Is Readrops the only one to have this behavior ?

I personally find logical to fetch only unread items the first time. The other times, Readrops fetches new items and updates the read state of items you already read. That makes sense to me. A mobile client doesn't need to have the same data as its web version.

I am not against fetching read and unread items at setup. But I think we should let the user decide whether this is what he wants. Would that be okay for you?

**Comment by xoxys** [comment]
I like the idea, configurable behavior would be even better. Thanks for your feedback!

**Comment by Alkarex** [comment]
A suggestion for default settings: not fetching read articles from the server during the global sync, but fetch them during a "pull to refresh" in a specific category / feed / label.

**Comment by SecT0uch** [comment]
I don't know if it's related, but I'm using the FreshRSS Web UI and other clients.

Here is the workflow:
* I sync with Readrops. It contains my unread article, say articleX
* I read articleX in the web UI
* I come back in Readrops and sync
=> articleX is still unread (in Readrops)

**Comment by Alkarex** [comment]
@SecT0uch That is another (related) issue, see e.g. https://github.com/readrops/Readrops/issues/52#issuecomment-594853944

**Comment by Mannshoch** [comment]
I read all article directly on Fresh but it seems that if I refresh readrop the app do not remove them from the app. Does that have the same cause?

**Comment by mgoral** [comment]
Not syncing unread articles on first sync also makes it harder to evaluate the app because it feels empty and users can't tell straight away if they like UI and the reading experience or not.

**Comment by Shinokuni** [comment]
@Mannshoch Read state sync is currently not available for FreshRSS.

@mgoral Do you mean read articles? An option will be available for fetching read and unread items at first sync. Otherwise, you can wait to get new items ;).

**Comment by Alkarex** [comment]
@Shinokuni Do not hesitate to ask for help regarding the API calls to implement those features

**Comment by Shinokuni** [comment]
Thanks for the proposal @Alkarex!

**Comment by Knusper** [comment]
> Yes, it is intended behavior. When syncing for the first time, Readrops fetches only unread items.

Why is this so. This is not really useful :(
